en/middleware/uxrce_dds.md

```diff
@@ -438,7 +439,96 @@ Each (`topic`,`type`) pairs defines:
 
 You can arbitrarily change the configuration.
-For example, you could use different default namespaces or use a custom package to store the message definitions. 
+For example, you could use different default namespaces or use a custom package to store the message definitions.
+
+## Fast-RTPS to uXRCE-DDS Migration Guidelines
+
+These guidelines explain how to migrate from using PX4 v1.13 [Fast-RTPS](../middleware/micrortps.md) middleware to PX4 v1.14 `uXRCE-DDS` middleware.
+These are useful if you have [ROS 2 applications written for PX4 v1.13](https://docs.px4.io/v1.13/en/ros/ros2_comm.html), or you have used Fast-RTPS to interface your applications to PX4 [directly](https://docs.px4.io/v1.13/en/middleware/micrortps.html#agent-in-an-offboard-fast-dds-interface-ros-independent).
+
+:::note
+This section contains migration-specific information.
+You should also read the rest of this page to properly understand uXRCE-DDS.
+:::
+
+
+#### Dependencies do not need to be removed
+
+uXRCE-DDS does not need the dependencies that were required for Fast-RTPS, such as those installed by following the topic [Fast DDS Installation](https://docs.px4.io/v1.13/en/dev_setup/fast-dds-installation.html).
+You can keep them if you want, without affecting your uXRCE-DDS applications.
+
+If you do choose to remove the dependencies, take care not to remove anything that is used by applications (for example, Java).
+
+#### `_rtps` targets are not needed
+
+The make targets with extension `_rtps` were used to build firmware that included client side RTPS code.
+
+The uXRCE-DDS middleware is included by default in builds for most boards, so you no longer need a special firmware to work with ROS 2. 
+To check if your board has it, look for `CONFIG_MODULES_UXRCE_DDS_CLIENT=y` in the `.px4board` file of your board.
+Those files are nested in [PX4-Autopilot/boards](https://github.com/PX4/PX4-Autopilot/tree/main/boards).
+
+If it is not present, or if it is set to `n`, then you have to clone the PX4 repo, modify the board configuration and manually [compile](../dev_setup/building_px4.md) the firmware.
+
+#### New client module and new start parameters
+
+As the client is implemented by a new PX4 module, you now have new parameters to start it.
+Take a look at the [client startup section](#starting-the-client) to learn how this is done.
+
+#### New file for setting which topics are published
+
+The list of topics that are published and subscribed for a particula firmware is now managed by the [dds_topic.yaml](https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/uxrce_dds_client/dds_topics.yaml) configuration file.
+
+See [Supported uORB Messages](#supported-uorb-messages) and [DDS Topics YAML](#dds-topics-yaml) sections for more information.
+
+
+#### Topics no longer need to be synced between client and agent.
+
+The list of bridged topics between agent and client no longer needs to be synced, so the `update_px4_ros2_bridge.sh` script is no longer needed.
```

Comment:
I don't really remember the detail for users here in terms of syncing. IF there is something we could link to that might be good.

Reply:
You only need to keep the message definitions in sync, but that is a ROS specific aspect

Reply:
```suggestion
The list of bridged topics between agent and client no longer needs to be synced for ROS 2, so the `update_px4_ros2_bridge.sh` script is no longer needed.
```